packages/react/bin/sync-fathom-events/index.ts

```diff
@@ -0,0 +1,63 @@
+import { writeFile } from "fs/promises";
```

Comment:
It would be cool to have this in a `scripts` folder. This is clearly not a binary.

Reply:
Fixed in bba31bed15c202f8cfeee7cafeaf0a70ef7395d6.